src/main/resources/db/migration/V24__structure_shinhan_point_plan_check_benefit.sql

```
@@ -0,0 +1,120 @@
+       'SHINHAN_POINT_PLAN_CHECK_CONVENIENCE', 'points', 'percentage', 'percent',
+       'flat', 'transaction_amount', 'standalone', 'point_accrual', 'transaction',
+       'direct', NULL, NULL, UTC_TIMESTAMP(6), UTC_TIMESTAMP(6)
+FROM card_benefits benefit
+INNER JOIN card_content_versions version
+    ON version.content_version_id = benefit.content_version_id
+INNER JOIN cards card ON card.card_id = version.card_id
+WHERE card.gorilla_card_id = '2890'
+  AND benefit.position = 5
+  AND benefit.benefit_id = '4da2cd93-b8e1-585c-bae4-7118aef652f8'
+  AND NOT EXISTS (
+      SELECT 1 FROM benefit_offers existing WHERE existing.benefit_id = benefit.benefit_id
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)